From 0718f1bb6e0a4ff8ff86dae2fa0172090b9247a8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Arthur=20Exalta=C3=A7=C3=A3o?=
 <arthur.exaltacao@chainguard.dev>
Date: Thu, 18 Dec 2025 13:47:52 -0300
Subject: [PATCH] fix: never complain about missing libraries
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This makes it so the build process doesn't check for libraries. ldd-check should do it for us

Signed-off-by: Arthur Exaltação <arthur.exaltacao@chainguard.dev>
---
 scripts/build_wheel.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/build_wheel.py b/scripts/build_wheel.py
index 03aae5861..c0230c9cc 100755
--- a/scripts/build_wheel.py
+++ b/scripts/build_wheel.py
@@ -357,6 +357,7 @@ def create_cuda_stub_links(cuda_stub_dir: str, missing_libs: list[str]) -> str:
 
 
 def check_missing_libs(lib_name: str) -> list[str]:
+    return []
     result = build_run(f"ldd {lib_name}", capture_output=True, text=True)
     missing = []
     for line in result.stdout.splitlines():
-- 
2.52.0

